pkg/server/proto/server.proto

```diff
@@ -564,6 +569,20 @@ message UI {
 
 message GetVersionInfoResponse {
   VersionInfo info = 1;
+
+  // Represents additional features that the current server supports.
+  ServerFeatures server_features = 2;
+}
+
+// Represents features that the server supports.
+// Useful for ensuring compatibility between clients and servers.
+message ServerFeatures {
+  repeated feature features = 1;
+
+  enum feature {
+    // Advertises that this server is capable of receiving inline keepalive messages
+    INLINE_KEEPALIVES = 0;
```

Comment:
For protobuf enums, [it is conventional](https://docs.buf.build/lint/configuration#enum_zero_value_suffix) to make the zero value "unspecified". It's [also convention](https://docs.buf.build/lint/rules#enum_value_prefix) to prefix the name of each value with the name of the enum. I don't know what the convention is within waypoint specifically, so if there is already an established convention, ignore me.
```suggestion
    FEATURE_UNSPECIFIED = 0
    // Advertises that this server is capable of receiving inline keepalive messages
    FEATURE_INLINE_KEEPALIVES = 1;
```

Reply:
Done! We don't have conventions around this as a team (that I'm aware of at least), and these conventions make sense to me. Yet another reason for us to eventually add `buf` to our process. Thank you!